tests/test_cppgc.rs

```diff
@@ -44,6 +44,10 @@ fn cppgc_object_wrap() {
       TRACE_COUNT.fetch_add(1, Ordering::SeqCst);
       visitor.trace(&self.value);
     }
+
+    fn get_name(&self) -> Option<&'static std::ffi::CStr> {
+      Some(c"Wrap")
```

Comment:
test somewhere that you can see this value?

Reply:
afaik there isn't really an easy way to test this since heap dumps need to be loaded into chrome to be read.

Reply:
maybe just see if it contains the byte string `Wrap` ?